.eslintignore

```diff
@@ -6,3 +6,5 @@ coverage
 /packages/bpk-component-icon/*/
 !/packages/bpk-component-icon/src
 /packages/bpk-tokens/tokens
+backpack-android
+backpack-ios
```

Comment:
is there a difference between `/backpack-android` and `backpack-android` ?

Just pointing out the inconsistency with the ignores above the lines you'e added.

Reply:
not sure, but it's not really inconsistent with the rest, everything except packages is not using the forward slash at the start 🤷‍♂️